be/src/common/status.cpp

```diff
@@ -261,6 +261,10 @@ std::string Status::code_as_string() const {
         return "Capaticy limit exceeded";
     case TStatusCode::SHUTDOWN:
         return "Shut down in progress";
+    case TStatusCode::BIG_QUERY_CPU_SECOND_LIMIT_EXCEEDED:
```

Comment:
do you have to handle big query memory limit?

Reply:
`big_query_mem_limit` is also handled by `TStatusCode:MEM_LIMIT_EXCEEDED`.